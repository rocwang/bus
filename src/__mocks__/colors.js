export default {
  oceanBlue: "#001120",
  green: "#8fc240",
  blue: "#00a3da",
  yellow: "#ffdb1b",
  red: "#da2f2d",
  orange: "#ea8c2e",
  black: "#000",
  grayA: "#333",
  grayB: "#666",
  grayC: "#999",
  grayD: "#ccc",
  white: "#fff",
};
