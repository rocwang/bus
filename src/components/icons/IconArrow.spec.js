import { mount } from "@vue/test-utils";
import IconArrow from "./IconArrow";
import colors from "../__fixtures__/colors";
import iconSpec from "./__fixtures__/icon.spec";

describe("IconArrow.vue", () => {
  iconSpec(IconArrow)();

  it("has a direction", () => {
    const wrapper = mount(IconArrow, {
      provide: {
        colors
      },
      propsData: {
        direction: "down"
      }
    });
    expect(wrapper.classes()).toContain(wrapper.vm.$style.down);
  });
});
