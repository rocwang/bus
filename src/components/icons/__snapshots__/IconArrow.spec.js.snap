// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`IconArrow.vue matches the snapshot 1`] = `
<svg
  viewBox="0 0 30 30"
  xmlns="http://www.w3.org/2000/svg"
>
  <path
    d="M2.01 17.245L.484 15.758a1.572 1.572 0 0 1 0-2.27L13.835.472c.646-.63 1.69-.63 2.33 0l13.35 13.01c.646.63.646 1.647 0 2.27l-1.525 1.486a1.681 1.681 0 0 1-2.357-.027L17.752 9.15v19.243c0 .89-.735 1.607-1.65 1.607h-2.198c-.914 0-1.65-.716-1.65-1.607V9.15l-7.887 8.068a1.67 1.67 0 0 1-2.357.027z"
    fill="#da2f2d"
  />
</svg>
`;
